cmd/util/ledger/migrations/deploy_migration.go

```diff
@@ -53,7 +54,7 @@ func NewTransactionBasedMigration(
 		return MergeRegisterChanges(
 			snapshot.Payloads,
 			executionSnapshot.WriteSet,
-			// we expect more  than one address to change here
+			expectedWriteAddress,
```

Comment:
does this pass tests? Since this is a transaction I expected it would also change some registers not on this address. For instance the global UUID register.

Reply:
Yeah, the test passes. However, I think you are right that this assumption that any deployment transaction only performs writes to the account where the contract is deployed to is not true, it just happens that the deployment of the `Burner` contract does. I'll refactor so only the Burner contract deployment gets checked that way.

What are other potential global registers that could be touched and might be worth double-checking?

Reply:
Refactored in [0e378d1](https://github.com/onflow/flow-go/pull/5479/commits/0e378d16517b26cb0ccc0f512cf43519df2ac7e1) and added test in [a50113a](https://github.com/onflow/flow-go/pull/5479/commits/a50113aebea7188a1b5c42c3a47a5076201e2e2d)

Reply:
> What are other potential global registers that could be touched and might be worth double-checking?

Besides the global uuid register I think it could also touch arbitrary other state depending on the contract **init** function.